vncdotool/client.py

```diff
@@ -347,7 +364,9 @@ def vncRequestPassword(self):
         self.sendPassword(self.factory.password)
 
     def vncConnectionMade(self):
-        self.setPixelFormat()
+        if not self.setImageMode():
```

Comment:
I'm finding this hard to parse. Would it make sense push setPixelFormat() inside setImageMode() to avoid the negative conditional?

Reply:
Yeah, that's better. Updated.